tests/ui-toml/min_ident_chars/clippy.toml

```diff
@@ -0,0 +1,2 @@
+allowed-idents-below-min-chars = ["Owo", "Uwu", "wha", "t_e", "lse", "_do", "_i_", "put", "her", "_e"]
```

Comment:
Looking at this list, I would think that we should allow indents starting with an underscore in general, as those are not used later on. I think it would be fine to allow them all the time, but you could also add a config value for that :)

Reply:
That sounds reasonable, I think not having a config value for it is fine